env.ts

```diff
@@ -0,0 +1,40 @@
+const getEnvValue = (
+  key: string,
+  value?: string,
+  nullable = false
+): string | undefined => {
+  if (typeof value === 'undefined' && !nullable) {
```

Comment:
```suggestion

const getEnvValue = (
  key: string,
  nullable = false
): string | undefined => {
	const value = process.env[key]
  	if (typeof value === 'undefined' && !nullable) {
  		throw new Error(`Missing env variable: ${key}`);
 	}
 	return value;
 }
 
const nonNullableEnvKeys = [
	'INSEE_CLIENT_ID_FALLBACK'
	...
];

const nullableEnvKeys = [
	'ALTERNATIVE_SEARCH_ROUTE',
	...
]
 
 const initializeEnvConfig = () => {
 	return [
 		nonNullableEnvKeys.map(getEnvValue)
 		nullableEnvKeys.map(k=>getEnvValue(k, true))
 	]
```

I think something like this would be simpler to update when adding a new env var. Just add a key in the relevant list.